frontend/package.json

```diff
@@ -45,7 +45,8 @@
     "use-debounce": "^8.0.4",
     "use-local-storage": "^3.0.0",
     "webfonts-loader": "^7.5.2",
-    "xss": "^1.0.14"
+    "xss": "^1.0.14",
+    "string-strip-html": "^13.4.3"
```

Comment:
How about using `xss` since we already have it in the bundle?  It can strip nodes completely with `stripIgnoreTagBody` option

Reply:
Done.